iChef-WebUi/.gitignore

```diff
@@ -46,3 +46,4 @@ Thumbs.db
 
 # Test Files
 Tests*.xml
+app-routing.module.ts
```

Comment:
Why is this ignored? @PinkyPotrick

Reply:
I wonder that as well, I told @mirunatodea not to commit the .gitignore, angular.json and package-lock.json files because we already had them uploaded and working on git.